examples/backlist_both_tokens.py

```diff
@@ -0,0 +1,48 @@
+from flask_jwt_extended import decode_token
+
+
+# Endpoint for loggin in
+@app.route('/login', methods=['POST'])
+def login():
+    username = request.json.get('username', None)
+    password = request.json.get('password', None)
+    if username != 'test' or password != 'test':
+        return jsonify({"msg": "Bad username or password"}), 401
+
+    user = {id: 1, username: 'test'}
```

Comment:
I think this would be more idomatic if we passed it in via the `@jwt.user_claims_loader` and accessing it later via `get_jwt_claims()`